src/pickup.cpp

```diff
@@ -98,6 +99,7 @@ static pickup_answer handle_problematic_pickup( const item &it, const std::strin
     amenu.text = explain;
 
     if( it.is_bucket_nonempty() ) {
+        amenu.addentry( WIELD, true, 'w', _("Wield %s"), it.display_name() );
```

Comment:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CODE_STYLE.md
```suggestion
        amenu.addentry( WIELD, true, 'w', _( "Wield %s" ), it.display_name() );
```